libparsec/src/validation.rs

```diff
@@ -0,0 +1,31 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use std::str::FromStr;
+
+/// Validate a label without doing a unicode normalization.
+pub fn validate_entry_name(raw: &str) -> bool {
+    libparsec_types::EntryName::is_valid(raw).is_ok()
```

Comment:
why not just `libparsec_types::EntryName::from_str(raw).is_ok()` ?

Reply:
I wanted to not have an allocation being made for validating an input, that's why I've added some `is_valid` that only take an `&str` and don't do the unicode normalization

Reply:
well shame the binding system is going to make an allocation anyway when converting the string from JS to Rust :trollface:

Reply:
I can change to use `from_str` if prefered in another PR, WDYT ?

Reply:
Don't bother that's not a big deal, just keep it in mind next time you're tempted with premature optimization ^^